src/pages/CustomerInvoicePage.js

```diff
@@ -165,7 +165,7 @@ export class CustomerInvoicePage extends GenericPage {
       [
         {
           title: `${pageInfoStrings.entry_date}:`,
-          info: formatDate(this.props.transaction.entryDate),
+          info: formatDate(this.props.transaction.entryDate) || 'N/A',
```

Comment:
Ancient history but you'd think the OG developers would have at least destructured this.props...

Reply:
9a5ebf0afdee319f322308af4cfea6829f2c5f5b 😄 .